lib/cmgd_bcknd_client.h

```diff
@@ -37,11 +37,13 @@ extern "C" {
 
 #define CMGD_BCKND_DEFAULT_CONN_RETRY_INTVL_SEC	5
 
-#define CMGD_BCKND_MSG_PROC_DELAY_MSEC		5
-#define CMGD_BCKND_MAX_NUM_MSG_PROC		100
+#define CMGD_BCKND_MSG_PROC_DELAY_MSEC		1
+#define CMGD_BCKND_MSG_PROC_DELAY_USEC		10
+#define CMGD_BCKND_MAX_NUM_MSG_PROC		500
 
-#define CMGD_BCKND_MSG_WRITE_DELAY_MSEC		50
-#define CMGD_BCKND_MAX_NUM_MSG_WRITE		1000
+#define CMGD_BCKND_MSG_WRITE_DELAY_MSEC		1
+#define CMGD_BCKND_MSG_WRITE_DELAY_USEC		10
```

Comment:
Please increase this to at least 1msec

Reply:
have replaced all write with CMGD_BCKND_MSG_WRITE_DELAY_MSEC which is 1msec